sql/telemetry_derived/fission_monitoring_crash_v1/query.sql

```diff
@@ -41,21 +61,41 @@ WITH crash_ping_data AS (
     `moz-fx-data-shared-prod.telemetry.crash`
   WHERE
     normalized_channel = 'nightly'
-    -- TODO: update filtering when https://bugzilla.mozilla.org/show_bug.cgi?id=1667426 is finalized
-    AND mozfun.map.get_key(environment.settings.user_prefs, 'fission.autostart') IS NOT NULL
+    AND mozfun.map.get_key(
+      environment.experiments,
+      "bug-1660366-pref-ongoing-fission-nightly-experiment-nightly-83-100"
+    ) IS NOT NULL
 ),
-main_ping_data AS (
+crash_ping_data AS (
+  SELECT
+    c.* EXCEPT (session_id)
+  FROM
+    crash_ping_data_unfiltered c
+  LEFT JOIN
+    first_sessions e
+  ON
+    c.session_id = e.session_id
+  WHERE
+    e.session_id IS NULL
+),
+main_ping_data_unfiltered AS (
   SELECT
     submission_timestamp,
     client_id,
-    -- TODO: update filtering when https://bugzilla.mozilla.org/show_bug.cgi?id=1667426 is finalized
+    payload.info.session_id,
     CASE
     WHEN
-      mozfun.map.get_key(environment.settings.user_prefs, 'fission.autostart') = 'true'
+      mozfun.map.get_key(
+        environment.experiments,
+        "bug-1660366-pref-ongoing-fission-nightly-experiment-nightly-83-100"
+      ).branch = 'fission-enabled'
```

Comment:
Same here as for crashes:

Could a check be added to ETL to ensure that the `mozfun.map.get_key(environment.settings.user_prefs, 'fission.autostart') == 'true'`, when `branch == 'enabled'` and vice-versa?

Reply:
We should use the new pref instead, we can add it to the crash table when it [lands in the schema](https://bugzilla.mozilla.org/show_bug.cgi?id=1669208), then we can look into checks.
I wouldn't delay this change though given the tight timeline.

Reply:
Sounds good.